src/App.js

```diff
@@ -36,29 +36,35 @@ class App extends Component {
     }
     return (
       <section>
-        <header>
-          <Link to='/'>
-          <h1><img className='logo' src={Tomatillos}/>Rancid Tomatillos</h1>
-          </Link>
-          <h2> Welcome {this.state.user.name || 'Movie Goer'}!</h2>
-            <Link to='/signin'>
-            <button type="button">Login</button>
+      <BrowserRouter>
```

Comment:
I would suggest rendering the BrowserRouter in the `index.js` file as a wrapper for the whole App. It gives us more flexibility when testing, and is a bit cleaner overall.